sql/src/main/java/io/crate/metadata/Functions.java

```diff
@@ -101,6 +72,37 @@ public FunctionImplementation get(String name, List<DataType> argumentsTypes) th
         return null;
     }
 
+    /**
+     * Returns the function implementation for the given function ident.
+     *
+     * @param ident The function ident.
+     * @return The function implementation.
+     * @throws UnsupportedOperationException if an implementation is not found.
+     */
+
+    public FunctionImplementation getQualified(FunctionIdent ident) throws IllegalArgumentException {
+        FunctionImplementation impl = getBuiltin(ident.name(), ident.argumentTypes());
+        if (impl == null) {
+            throw createUnknownFunctionException(null, ident.name(), ident.argumentTypes());
+        }
+        return impl;
+    }
+
+    public static UnsupportedOperationException createUnknownFunctionException(@Nullable RuntimeException e,
```

Comment:
@mfussenegger there is: https://github.com/crate/crate/pull/5256/files/c1cb7e32e11c4f1a93949fcc0e117de55758a7ba#diff-122c946a537ea2e310a4af0ca4bd0ae7R86